crawl-ref/source/status.cc

```diff
@@ -962,6 +962,12 @@ static void _describe_gem(status_info& inf)
     if (time_taken >= limit)
         return; // already lost...
 
+    if (!gem_clock_active())
+    {
```

Comment:
I think you also want to check that the player doesn't yet have the gem. As currently written, "Gem not acquired" will always be visible, even when the player has the gem!